subsys/lorawan/lorawan.c

```diff
@@ -102,7 +102,7 @@ static void datarate_observe(bool force_notification)
 	LoRaMacMibGetRequestConfirm(&mibGet);
 
 	if ((mibGet.Param.ChannelsDatarate != current_datarate) ||
-	    (force_notification)) {
+		(force_notification)) {
```

Comment:
May I know why you've changed this? The wrapped line was intentionally aligned with the opening bracket `(`

Reply:
I think my IDE wanted to make it more fancy. I'll revert it